lib/puppet/type/group.rb

```diff
@@ -36,6 +36,9 @@ module Puppet
       newvalue(:absent) do
         provider.delete
       end
+
+    defaultto :present
```

Comment:
@suckatrash thanks for updating this! The indentation on this line is still off though. You just need to add two spaced before `defaultto`. Thanks!